src/componentes/CardList/CardList.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import ratings from '../../ratings.json';
+import Card from "../card/card";
+
+function CardList(props) {
+
+  const cardMap = ratings.map( 
+    (rating) => <Card rating={rating}></Card>
```

Comment:
Para evitar confusiones adentro del componente `<Card/>`, lo mejor sería cambiar el nombre de la prop a uno que evite repetir tantas veces la palabra `rating`.

```suggestion
    (local) => <Card local={local}></Card>
```